packages/application/src/plugins/rise.ts

```diff
@@ -1217,3 +1220,49 @@ namespace Rise {
     return reveal_helpstr;
   }
 }
+
+namespace Private {
```

Comment:
i was bitten by TypeScript 😜

```suggestion
// @ts-expect-error 'Private' may never be read
namespace Private {
```